src/node.js

```diff
@@ -131,9 +131,16 @@
         if (process._forceRepl || NativeModule.require('tty').isatty(0)) {
           // REPL
           var cliRepl = Module.requireRepl();
-          cliRepl.createInternalRepl(process.env, function(err, repl) {
+          cliRepl.createInternalRepl(process.env, true, function(err, repl) {
             if (err) {
-              throw err;
+              console.log('Encountered error with persistent history support.');
+              return cliRepl.createInternalRepl(
+                process.env, false, function(err, repl) {
```

Comment:
The linter is set up pretty conservative right now. Not sure it has a rule for odd linebreaks, but it seems fine to me there.